Firestore/Source/Local/FSTLRUGarbageCollector.mm

```diff
@@ -16,17 +16,31 @@
 
 #import "Firestore/Source/Local/FSTLRUGarbageCollector.h"
 
+#include <chrono> //NOLINT(build/c++11)
 #include <queue>
+#include <utility>
 
 #import "Firestore/Source/Local/FSTMutationQueue.h"
+#import "Firestore/Source/Local/FSTPersistence.h"
 #import "Firestore/Source/Local/FSTQueryCache.h"
+#include "Firestore/core/include/firebase/firestore/timestamp.h"
 #include "Firestore/core/src/firebase/firestore/model/document_key.h"
+#include "Firestore/core/src/firebase/firestore/util/log.h"
 
+using Millis = std::chrono::milliseconds;
+using firebase::Timestamp;
+using firebase::firestore::local::LruParams;
+using firebase::firestore::local::LruResults;
 using firebase::firestore::model::DocumentKey;
 using firebase::firestore::model::ListenSequenceNumber;
 
+const long kFIRFirestorePersistenceCacheSizeUnlimited = -1;
```

Comment:
Shouldn't this declaration should be up somewhere in API-land? Eventually this code will be platform neutral C++, while this constant has a name that's specific to Objective-C.